components/api_request_helper/api_request_helper.h

```diff
@@ -100,6 +101,9 @@ class APIRequestHelper {
   // into string before validating the response. For these purposes
   // conversion_callback is added which receives raw response and can perform
   // necessary conversions.
+  //
+  // This method will be deprecated soon,
```

Comment:
// This method will be deprecated soon, please use the one underneath with APIRequestOption parameter.
  // https://github.com/brave/brave-browser/issues/29611